SimplePolicy/SimplePolicyPlugin.cs

```diff
@@ -1,83 +1,14 @@
     {
         private static readonly string log_dictionary = Path.Combine(AppContext.BaseDirectory, "Logs");
 
-        public override void Configure()
-        {
-            Settings.Load(GetConfiguration());
-        }
-
-        public bool FilterForMemoryPool(Transaction tx)
-        {
-            if (!VerifySizeLimits(tx)) return false;
-
-            switch (Settings.Default.BlockedAccounts.Type)
-            {
-                case PolicyType.AllowAll:
-                    return true;
-                case PolicyType.AllowList:
-                    return tx.Witnesses.All(p => Settings.Default.BlockedAccounts.List.Contains(p.VerificationScript.ToScriptHash()));
-                case PolicyType.DenyList:
-                    return tx.Witnesses.All(p => !Settings.Default.BlockedAccounts.List.Contains(p.VerificationScript.ToScriptHash()));
-                default:
-                    return false;
-            }
-        }
-
-        public IEnumerable<Transaction> FilterForBlock(IEnumerable<Transaction> transactions)
-        {
-            return FilterForBlock_Policy2(transactions);
-        }
-
-        public int MaxTxPerBlock => Settings.Default.MaxTransactionsPerBlock;
-        public int MaxLowPriorityTxPerBlock => Settings.Default.MaxFreeTransactionsPerBlock;
-
-        private static IEnumerable<Transaction> FilterForBlock_Policy1(IEnumerable<Transaction> transactions)
-        {
-            int count = 0, count_free = 0;
-            foreach (Transaction tx in transactions.OrderByDescending(p => p.NetworkFee / p.Size).ThenByDescending(p => p.NetworkFee))
-            {
-                if (count++ >= Settings.Default.MaxTransactionsPerBlock - 1) break;
-                if (!tx.IsLowPriority || count_free++ < Settings.Default.MaxFreeTransactionsPerBlock)
-                    yield return tx;
-            }
-        }
-
-        private static IEnumerable<Transaction> FilterForBlock_Policy2(IEnumerable<Transaction> transactions)
-        {
-            if (!(transactions is IReadOnlyList<Transaction> tx_list))
-                tx_list = transactions.ToArray();
-
-            Transaction[] free = tx_list.Where(p => p.IsLowPriority)
-                .OrderByDescending(p => p.NetworkFee / p.Size)
-                .ThenByDescending(p => p.NetworkFee)
-                .ThenBy(p => p.Hash)
-                .Take(Settings.Default.MaxFreeTransactionsPerBlock)
-                .ToArray();
-
-            Transaction[] non_free = tx_list.Where(p => !p.IsLowPriority)
-                .OrderByDescending(p => p.NetworkFee / p.Size)
-                .ThenByDescending(p => p.NetworkFee)
-                .ThenBy(p => p.Hash)
-                .Take(Settings.Default.MaxTransactionsPerBlock - free.Length - 1)
-                .ToArray();
-
-            return non_free.Concat(free);
-        }
-
         void ILogPlugin.Log(string source, LogLevel level, string message)
         {
             if (source != nameof(ConsensusService)) return;
```

Comment:
this should be moved to ApplicationLogs

Reply:
Maybe we should create a new plugin to categorize and save all the logs, not just the consensus logs. I think ApplicationLogs is not used to do these logging, it is only used to record the results of the Application execution.